pkg/codegen/codegen.go

```diff
@@ -233,8 +233,12 @@ func Generate(spec *openapi3.T, opts Configuration) (string, error) {
 	}
 
 	var inlinedSpec string
+	var refNameResolver openapi3.RefNameResolver
+	if opts.OutputOptions.InternalizeRefs {
+		refNameResolver = openapi3.DefaultRefNameResolver
+	}
```

Comment:
Unfortunately this changes the API so isn't compatible and would need a major version increment. See `inline.go` comment for how to solve this.
```suggestion
```